src/cljs/witan/ui/schema.cljs

```diff
@@ -11,6 +11,14 @@
    :kixi.group/type   s/Keyword
    :kixi.group/id     uuid?})
 
+(def UserSchema
+  {:kixi.user/name   (s/maybe s/Str)
+   :kixi.user/id     (s/maybe s/Str)
```

Comment:
The transmission of UUIDs is a pain and in most cases we're only sending it back to the backend anyway, so no need to convert